Dockerfile.dev

```diff
@@ -1,16 +1,7 @@
-FROM node:14 AS base
+FROM node:14-alpine
```

Comment:
have you verified that the alpine image works?

Reply:
Yea @gmbronco has been using it for development. I just tried it then and it worked for me.